src/components/footer/footer.tsx

```diff
@@ -92,17 +83,32 @@ export const Footer = ({ className }: FooterProps) => {
                 </ul>
                 <ul>
                     <li>
-                        <Link to={ROUTES.home.to()} className={styles.navItem}>
+                        <Link
+                            to="https://www.facebook.com/WixStudio"
+                            className={styles.navItem}
+                            target="blank"
+                            rel="noreferrer"
```

Comment:
Why noreferrer?

Reply:
We don't  know if we can trust the site user wants to link to. I think lets keep referrer private.

Reply:
There are a lot of resources which advise to add this attribute. All links in Codux with target="_blank" uses it.
I also cannot remove it in the project without [adding eslint rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#rule-details)
<img width="840" alt="Screenshot 2024-09-17 at 09 20 26" src="https://github.com/user-attachments/assets/1fa7886e-e860-4dae-b94a-93091cdaee78">

Reply:
> There are a lot of resources

Those lots of resources at at least 10 years old and apply only to Internet Explorer.

> I also cannot remove it in the project without [adding eslint rule]

Not adding, removing. It's already added and it's completely useless.

> We don't know if we can trust the site user wants to link to. I think lets keep referrer private.

So this has nothing to do with the trust.